### ISSUE TITLE
FlowRouter.go("/") doesn't change url

### ISSUE THREAD

**Issue by copleykj** [meteorhacks/flow-router#147]
If application is on a route such as /posts and you use FlowRouter.go("/") to navigate to the root of your application, the page for root renders correctly but the URL in the location bar never changes.

**Comment by arunoda** [comment]
Seems like an issue with how you use FlowLayout. You must share some sample code, otherwise we can't see what's going on there.

**Comment by copleykj** [comment]
Very sorry for this.. I'm not sure exactly why this happened but I created a reproduction which seemed to work flawlessly. I then removed parts of the non working app bit by bit till it matched the working reproduction exactly. At that point it still didn't work so I deleted the .meteor directory from the non working app and copied the one from the working app and then it worked perfectly.. UGH! Oh well I guess I'll chock this one up to a strange glitch.

**Comment by copleykj** [comment]
So apparently when I used meteorpad to reproduce this issue it was using older versions of flow-router and flow-layout. Not realizing this I thought that the issue was resolved by moving the .meteor directory but that was just a symptom of the dependencies being downgraded to the versions the reproduction was using.

I've created a reproduction of the issue here: https://github.com/copleykj/FlowRouterReproduction. In the reproduction it has 3 templates each with a link that when clicked calls FlowRouter.go() to move the the subsequent template. first click moves from template1 to template2 and you see /template2 in the url, second click moves from template2 to template3 and you see /template3 in the path. Third click moves back to template1 which is located at the root but the path stays at /template3 instead of /
